pkg/katautils/config.go

```diff
@@ -994,6 +993,17 @@ func SetKernelParams(runtimeConfig *oci.RuntimeConfig) error {
 		}
 	}
 
+	// set the scsi scan mode to none for virtio-scsi
+	if runtimeConfig.HypervisorConfig.BlockDeviceDriver == config.VirtioSCSI {
+		p := vc.Param{
+			Key:   "scsi_mod.scan",
+			Value: "none",
+		}
+		if err := (runtimeConfig).AddKernelParam(p); err != nil {
```

Comment:
s/(runtimeConfig)/runtimeConfig

Reply:
ok， I will also s/(runtimeConfig)/runtimeConfig globally in pkg/katautils/config.go